docs/kpod-login.1.md

```diff
@@ -0,0 +1,67 @@
+% kpod(1) kpod-login - Simple tool to login to a registry server
+% Urvashi Mohnani
+# kpod-login "1" "August 2017" "kpod"
+
+## NAME
+kpod-login - Login to a container registry
+
+## SYNOPSIS
+**kpod login**
+**SERVER**
+[**--help**|**-h**]
+
+## DESCRIPTION
+**kpod login** logs into a specified registry server with the correct username
+and password. **kpod login** reads in the username and password from STDIN.
+The username and password can also be set using the **username** and **password** flags.
+The path of the authentication file can be specified by the user by setting the **override-path**
+flag. The default path used is **XDG\_RUNTIME\_DIR/containers/**.
```

Comment:
This is a directory.  Should be **XDG\_RUNTIME\_DIR/containers/REGISTRY.json**.

Reply:
fixed